README.md

```diff
@@ -110,6 +110,26 @@ new Fingerprint2({
 });
 ```
 
+#### `preprocessor` option
+
+Function that is called with each component value that may be used to modify component values before computing the fingerprint.
+For example: strip browser version from user agent.
+
+```
+new Fingerprint2({
+  preprocessor: function(key, value){
+    if(key == "user_agent"){
+      var parsedUserAgent = SomeUserAgentParser.parse(value);
+      var userAgentMinusVersion = parsedUserAgent.base_platform + ' ' + parsedUserAgent.browser_name;
+      return userAgentMinusVersion;
```

Comment:
I think the example is a bit hard to understand. Could we either

- simplify the example, maybe using regex substitution for the version number (not sure if such regex exists?)
- or use a concrete user agent library (just the real API, no imports etc.)